sdk/src/main/java/com/akkaserverless/javasdk/eventsourcedentity/EventSourcedEntity.java

```diff
@@ -94,9 +94,7 @@ public final void _internalSetEventContext(Optional<EventContext> context) {
 
       OnSuccessBuilder<S> emitEvent(Object event);
 
-      OnSuccessBuilder<S> emitEvents(Object event1, Object... additionalEvents);
-
-      OnSuccessBuilder<S> emitEvents(Collection<? extends Object> event);
+      OnSuccessBuilder<S> emitEvents(List<?> event);
```

Comment:
Not good to allow unordered things here, such as `Set`.

Also, I think there is a problem with overloads with vararg Object like that.